setup.py

```diff
@@ -22,7 +22,7 @@
         'paramiko==2.4.2',
         'pyyaml>=4.2b1',
         'jinja2==2.10.1',
-        'kubernetes==10.0.1',  # latest openshift requires this version
+        'kubernetes~=10.0.1',  # latest openshift requires this version
```

Comment:
>= for better readability but one and the same.

Reply:
>= would give us 11.x.x which was causing the original issue.

Reply:
No, that's not the same:
* `~=10.0.1` will match 10.0.1, 10.0.2 etc, but not 10.1.0 not 11.0.0, etc.
* `>=10.0.1` will match 10.0.1, 10.0.2, 10.1.0, 11.0.0.  ==> This fails here, because we get 11.0.0 in this case. And openshift requires `~= 10.0.1`.

Reply:
see #1644

Reply:
I see, my initial thought was with major fixed >=10.0.* it would still stick with version, but i guess we need both '>=10.0.1,<=11'